project_scrum/models/project_scrum_sprint.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+# Copyright <2017> <Tenovar Ltd>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models, fields, api
+from datetime import date, timedelta
+
+
+class ProjectScrumSprint(models.Model):
+    _name = 'project.scrum.sprint'
+    _description = 'Project Scrum Sprint'
+    _order = 'date_start desc'
+
+    def get_current_sprint(self, project_id):
+        scrum_obj = self.env['project.scrum.sprint']
+        sprint = scrum_obj.search([
+            '&', '&',
+            ('date_start', '<=', date.today()),
+            ('date_stop', '>=', date.today()),
+            ('project_id', '=', project_id)
+        ])
+        if sprint:
+            return sprint[0]
```

Comment:
set `limit=1` above instead. actually, you can just return the search with the limit, it will be an emtpy recordset if the search is empty